go.mod

```diff
@@ -5,6 +5,7 @@ go 1.20
 require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/bitly/go-simplejson v0.5.0
+	github.com/envoyproxy/go-control-plane v0.10.3
```

Comment:
what is this being added for? I couldn't find where it was being used, but I might have missed it.

Reply:
Don't remember, but it's definitely not used 👀 removed it!